packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2193,6 +2193,23 @@ class ShaderMaskLayer extends ContainerLayer {
   }
 }
 
+/// A backdrop key uniquely identifies the backdrop that a [BackdropFilterLayer]
+/// samples from.
+///
+/// When multiple backdrop filters share the same key, the Flutter engine can
+/// more efficiently perform the backdrop operations.
+///
+/// For more information, see `BackdropFilter`.
+@immutable
+final class BackdropKey {
+  /// Create a new [BackdropKey].
+  BackdropKey() : _id = _nextId++;
+
+  static int _nextId = 0;
+
+  final int _id;
```

Comment:
We should probably call these "key", not "id"s to remain consistent.

Reply:
Done